platform-sdk/sdk/settings.txt

```diff
@@ -10,10 +10,13 @@
                                           #############################
                                           #    Gossip / Networking    #
                                           #############################
-
-chatter.useChatter,                       false
 sync.syncProtocolPermitCount,             2
 
+                                          #############################
+                                          #           State           #
+                                          #############################
+state.saveStatePeriod,                    60

```

Comment:
Q: should this be 900?
I'm looking at https://github.com/hashgraph/hedera-services/pull/9341/files#diff-64ffea1d3c829bdd759986b8faf13e821cc95918a7ce7bc6f8286d87124f55ecL15

Reply:
This file is used when running locally. Usually when we run locally it is much better to save a state every minute than 15 minutes.